pygeo/parameterization/DVGeo.py

```diff
@@ -1686,22 +1686,22 @@ def update(self, ptSetName, childDelta=True, config=None):
         # We've postponed things as long as we can...do the finalization.
         self._finalize()
 
-        for iChild in range(len(self.children)):
-            if len(self.children[iChild].axis) > 0:
-                self.children[iChild]._finalize()
-                refaxis_ptSetName = "child%d_axis" % (iChild)
-                if refaxis_ptSetName not in self.FFD.embeddedVolumes:
-                    print("adding refaxis_ptSetName", refaxis_ptSetName)
-                    self.FFD.attachPoints(self.children[iChild].refAxis.coef, refaxis_ptSetName)
-                    self.FFD.calcdPtdCoef("child%d_axis" % (iChild))
-
         # Make sure coefficients are complex
         self._complexifyCoef()
 
         # Set all coef Values back to initial values
         if not self.isChild:
             self.FFD.coef = self.origFFDCoef.copy()
             self._setInitialValues()
+            
+            for iChild in range(len(self.children)):
```

Comment:
Good catch here, did not realize that the children were trying to initialize themselves again. Out of curiosity, did you have issues because of this or you just re-checked the code?